### DISCUSSION TITLE
Duplicate entries

### DISCUSSION THREAD

**Discussion by cgraham149** [navidrome/navidrome#3358] [Q&A] [answered]
I can't for the life of me figure out how these duplicate entries are occurring. There are only single files in the directory but it shows duplicate song entries. I've combed through the metatags, rescanned the directory, and restarted the docker container but these duplicates persist. Any ideas? Is there a way to flush the cache or something?

FYI - they appear as they should in Emby, Clementine, Picard, and Kid3.

![image](https://github.com/user-attachments/assets/d717d91f-efa1-4cdf-98fd-9a1504738f60)

**Comment by deluan** [comment]
Things to check:
1. Logs. Run a Quick Scan and see if it shows any errors
2. Check if the paths for the duplicated songs is the same:

https://github.com/user-attachments/assets/2f98d42f-6725-4889-9b1c-08526b27ee22

**Reply by cgraham149** [reply]
The Get Info shows an invalid path for each duplicate and a correct path for the right file. Running either a  Quick scan or Full scan does not resolve the issue.

**Reply by deluan** [reply]
Check the logs. Probably the scan process is being aborted before it completes, leaving old entries in place. The logs should say what is the error if this is the case.

**Reply by cgraham149** [reply, marked as answer]
I ended up deleting the cache files in my persistent data mount for the docker container. It is now resolved.

**Reply by joeis-dev** [reply]
Did you just delete everything inside /data/cache manually?
